Gridy/View/GridFrameView.swift

```diff
@@ -17,11 +17,11 @@ class GridFrameView: UIView {
         super.init(coder: coder)
     }
     
-    func getX() ->CGFloat {
+    static func getX() ->CGFloat {
         return self.frame.origin.x
```

Comment:
You probably had an error from you compiler, but you can't use instance methods or properties in class (static method)